src/internal_modules/roc_packet/sorted_queue.cpp

```diff
@@ -38,7 +38,8 @@ void SortedQueue::write(const PacketPtr& packet) {
                 "sorted queue: queue is full, dropping packet:"
                 " max_size=%u",
                 (unsigned)max_size_);
-        return;
+        // TODO: return StatusAgain (gh-183)
+        return status::StatusNoData;
```

Comment:
It shouldn't be StatusAgain. In this case upper code will immediately retry writing this packet. But we don't want it, if queue is full, we really want to drop packet. So we should drop and return OK.